pandas/io/json/_table_schema.py

```diff
@@ -147,8 +147,7 @@ def convert_pandas_type_to_json_field(arr) -> dict[str, JSONSerializable]:
             # timezone.utc has no "zone" attr
             field["tz"] = "UTC"
         else:
-            # error: "tzinfo" has no attribute "zone"
-            field["tz"] = dtype.tz.zone  # type: ignore[attr-defined]
+            field["tz"] = str(dtype.tz)
```

Comment:
We probably don't have good test coverage for this but are you sure that the string representation of `dtype.tz` is the same as the `.zone` attribute prior?